deploy/single-v2/all-in-one-postgres.yaml

```diff
@@ -44,96 +43,42 @@ spec:
-              internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
-            type: string
           config:
-            description: Config contains the plugin configuration.
             type: object
             x-kubernetes-preserve-unknown-fields: true
           configFrom:
-            description: ConfigFrom references a secret containing the plugin configuration.
             properties:
-              apiVersion:
-                description: 'APIVersion defines the versioned schema of this representation
-                  of an object. Servers should convert recognized schemas to the latest
-                  internal value, and may reject unrecognized values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
-                type: string
-              kind:
-                description: 'Kind is a string value representing the REST resource
-                  this object represents. Servers may infer this from the endpoint
-                  the client submits requests to. Cannot be updated. In CamelCase.
-                  More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
-                type: string
               secretKeyRef:
-                description: NamespacedSecretValueFromSource represents the source
-                  of a secret value specifying the secret namespace
                 properties:
-                  apiVersion:
-                    description: 'APIVersion defines the versioned schema of this
-                      representation of an object. Servers should convert recognized
-                      schemas to the latest internal value, and may reject unrecognized
-                      values. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#resources'
-                    type: string
                   key:
-                    description: the key containing the value
-                    type: string
-                  kind:
-                    description: 'Kind is a string value representing the REST resource
-                      this object represents. Servers may infer this from the endpoint
-                      the client submits requests to. Cannot be updated. In CamelCase.
-                      More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
                     type: string
                   name:
-                    description: the secret containing the key
                     type: string
                   namespace:
-                    description: The namespace containing the secret
                     type: string
                 required:
-                - key
                 - name
                 - namespace
+                - key
```

Comment:
Could you please help to revert to previous order since there is no actual changes ?
key
name
namespace

Reply:
Shouldn't really matter, as it makes no difference to end users. The change in order is just an artifact of the original 1.x CRDs not sorting array items alphabetically, whereas Kubebuilder generation does sort. I don't think we need to re-order the originals to match since it's just a cosmetic thing that affects the diff display.

Reply:
The changed scope in this PR is very large and confused, follow previous order making detect changes easier since it is not a real change.

Reply:
Choose to leave as it is since this might due to GitHub Diff annoying.